src/features/user/MFV2/ContributionsMap/Markers/ClusterMarker.tsx

```diff
@@ -0,0 +1,102 @@
+import { Marker } from 'react-map-gl-v7';
+import { MutableRefObject, useEffect, useState } from 'react';
+import { PointFeature, AnyProps } from 'supercluster';
+import { _getClusterGeojson } from '../../../../../utils/superclusterConfig';
+import { useMyForestV2 } from '../../../../common/Layout/MyForestContextV2';
+import ClusterIcon from './ClusterIcon';
+import {
+  ProjectPurposeTypes,
+  TreeProjectClassification,
+  UnitTypes,
+} from '@planet-sdk/common';
+import {
+  getClusterMarkerColors,
+  extractAndClassifyProjectData,
+} from '../../../../../utils/myForestV2Utils';
+import { ViewportProps } from '../../../../common/types/map';
+
+export interface ClusterMarkerProps {
+  superclusterResponse: PointFeature<AnyProps>;
+  viewport: ViewportProps;
+  mapRef: MutableRefObject<null>;
+}
+
+export type ExtractedData = {
+  unitType: UnitTypes;
+  classification: TreeProjectClassification;
+  purpose: ProjectPurposeTypes;
+  contributionCount: number;
+};
```

Comment:
The name `ExtractedData` does not give a clear idea about the purpose. How can this be named more relevantly?

Reply:
e.g. `ExtractedProjectData`